source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -36,6 +36,47 @@ You can create a Prometheus service within your PaaS space and allow it to bind
 
 Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana](https://grafana-paas.cloudapps.digital).
 
+### Container metrics
+
+Cloud Foundry provides time-series data, or metrics, for each instance of your PaaS app. You should use the metrics exporter app to send this data to a monitoring system of your choice. Your monitoring system can then store this data over time and let you view it.
+
+To use the metrics exporter, you deploy it as an app on PaaS. The current metrics supported by the metrics exporter app are CPU, RAM, disk usage data, app crashes, app requests and app response times.
+
+### Setting up the metrics exporter app
+
+Before you set up the metrics exporter app, you will need a live Cloud Foundry account assigned to the spaces you want to receive metrics on; this account should be separate to your primary Cloud Foundry account.
+
+We recommend that the separate Cloud Foundry account uses the [`SpaceAuditor` role](https://docs.cloud.service.gov.uk/#user-roles-space-auditor), as this role has the minimum permissions needed to meet the requirements of the metrics exporter app.
+
+To set up the metrics app:
+
+1. Clone the [https://github.com/alphagov/paas-metric-exporter](https://github.com/alphagov/paas-metric-exporter) repository.
+2. [Push the metrics exporter app](https://docs.cloud.service.gov.uk/#deployment-overview) to Cloud Foundry without starting the app by running `cf push --no-start metric-exporter`.
+3. Set the following mandatory environment variables in the metrics exporter app using `cf set-env metric-exporter NAME VALUE`:
+
+	|Name|Value|
+	|:---|:---|
+	|`API_ENDPOINT`|Use `https://api.cloud.service.gov.uk`|
+	|`USERNAME`|Cloud Foundry User|
+	|`PASSWORD`|Cloud Foundry Password|
+	|`ENABLE_STATSD`|false|
+	|`ENABLE_PROMETHEUS`|true|
+
+	You should use the `cf set-env` command for these mandatory variables as they contain secret information, and this method will keep them secure.
+
+	You can also set environment variables by amending the manifest file. We recommend that you use this method for optional environment variables that do not contain secret information. Refer to the [https://github.com/alphagov/paas-metric-exporter](https://github.com/alphagov/paas-metric-exporter) repository for more information.
+
+4. Start your app by running `cf start metric-exporter`.
+
```

Comment:
Should also include information about how to secure the container metrics endpoint as teams may be concerned that the container metrics will expose information about the underlying infrastructure.